### DISCUSSION TITLE
Is there a way to show 2 price lines on the same pane?

### DISCUSSION THREAD

**Discussion by Arnau-Gelonch** [klinecharts/KLineChart#532] [Q&A]
Does anyone know if it is possible to implement something like this:
![image](https://github.com/klinecharts/KLineChart/assets/62238598/2f94a53f-a789-4f1a-973b-f86ddfee48af)

**Comment by ct152098** [comment]
I have the same problem, too

**Comment by icrefin** [comment]
// Create first price line  
chart.createOverlay({  
  name: 'priceLine',  
  points: [{ value: 100.50 }]  // First price level  
});  
  
// Create second price line  
chart.createOverlay({  
  name: 'priceLine',  
  points: [{ value: 105.75 }]  // Second price level  
});
